orderedmap/builder.go

```diff
@@ -36,6 +36,9 @@ type findValueUntyped interface {
 // ToYamlNode converts the ordered map to a yaml node ready for marshalling.
 func (o *Map[K, V]) ToYamlNode(n NodeBuilder, l any) *yaml.Node {
 	p := utils.CreateEmptyMapNode()
+	if o != nil {
```

Comment:
Can you add a small test for this, as this new block will reduced coverage, if you pull the project again and update the workflow config (upgraded the codecov action) You should see the coverage on this PR.

Reply:
I merged the latest change from main branch and ran `go test -v -coverprofile cover.out ./...` locally and see 100% coverage.  In the status on this PR I see `1 workflow awaiting approval`, which I assume means you need to approve something before the coverage check can run here, but I'm unfamiliar.  I'd appreciate if you clarify for me.

Reply:
whoops, nevermind I was reading the test output incorrectly.  I'll try to figure it out.